### PR TITLE
chore: add examples for agent article

### PR DESCRIPTION
**Pull request by olbychos**
- add examples just for article purposes

### CODE REVIEW

File: examples/use_case_project_manager/app.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    with st.chat_message("assistant"):
+        message_placeholder = st.empty()
+        full_response = ""
+
+        st.session_state.messages.append({"role": "assistant", "content": ""})
+
+        for chunk in generate_agent_response(st.session_state.agent, user_input):
+            full_response += chunk
+            message_placeholder.markdown(full_response + "▌")
```
Comment (MaksymAI) [inline examples/use_case_project_manager/app.py:30]: I am curious about what this symbol (`▌`) is and why we need it?))

File: examples/use_case_project_manager/composio_tool.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        field_definitions[field_name] = Field(**field_kwargs)
+
+    # Create the namespace for the class
+    namespace = {"__annotations__": annotations, **field_definitions}
+
+    # Create the model class
+    return type(model_name, (BaseModel,), namespace)
+
+
+class ComposioTool(Node):
```
Comment (MaksymAI) [inline examples/use_case_project_manager/composio_tool.py:57]: Should we move this tool from the examples section to the tools section?

**Review by MaksymAI — APPROVED** [review]

File: examples/use_case_project_manager/composio_tool.py (hunk 2 above)
Comment (olbychos) [reply examples/use_case_project_manager/composio_tool.py:57]: We will keep this tool in our separate repository along with the dynamiq tools. This arrangement is temporary and is intended for article publication.

File: examples/use_case_project_manager/app.py (hunk 1 above)
Comment (olbychos) [reply examples/use_case_project_manager/app.py:30]: Yes, it's for using a delimiter in Streamlit to manage outputs.

**Pull request merged**